packages/insomnia-inso/src/commands/run-tests.js

```diff
@@ -19,38 +18,65 @@ export const TestReporterEnum = {
 export type RunTestsOptions = GlobalOptions & {
   env?: string,
   reporter: $Keys<typeof TestReporterEnum>,
```

Comment:
I believe this should be typed as string now

```suggestion
  reporter: string,
```